sourcegraph/templates/prometheus/prometheus.ConfigMap.yaml

```diff
@@ -236,7 +241,7 @@ data:
       # target_label: kubernetes_namespace
       - source_labels: [__meta_kubernetes_namespace]
         action: replace
-        target_label: {{ .Release.Namespace }}
+        target_label: ns
```

Comment:
Was this intended?

Reply:
Ah! Yes, this fixes an earlier mistake on my part - this should never have been a dynamic value, just a label name.

Reply:
Actually I think I see here ... `target_label` not the value...